### ISSUE TITLE
The code of the classification part of the graph seems to be inconsistent with that described in formula 4.2 of the paper

### ISSUE THREAD

**Issue by wind603** [weihua916/powerful-gnns#26]
Hello, thank you for your great work!

The code does not seem to be as described in the paper, concat the graph level features of each layer and then pass through the classification layer.
It is the sum of the scores of the graph level features of each layer obtained by the classification layer.

below is the code in 'class GraphCNN':

       #perform pooling over all nodes in each graph in every layer
        for layer, h in enumerate(hidden_rep):
            pooled_h = torch.spmm(graph_pool, h)
            score_over_layer += F.dropout(self.linears_prediction[layer](pooled_h), self.final_dropout, training = self.training)

        return score_over_layer

**Comment by jorel666** [comment]
Hi @weihua916! Hi @weihua916 — 12 AI repos? ARKHAN adds 59 more autonomous chips: scraping, outreach, crypto, delivery.

Starter: $6.50 — https://xipetotec.lemonsqueezy.com/checkout/buy/8f4322fc-f700-456c-8278-241ba4307ae6
Full API: https://xipetotec.lemonsqueezy.com/checkout/buy/d5943e3e-b661-4011-a995-826847b87156

Reply STOP to unsubscribe.

Pago directo USDT TRC20: TWJv7sD1nTWeHFsaMDBuJEEAc62VoxPmvm
Ref: pay_99d396992006

*Automated outreach — reply NO to stop.*
